cache4k/src/commonTest/kotlin/io/github/reactivecircus/cache4k/CacheListenerTest.kt

```diff
@@ -0,0 +1,147 @@
+package io.github.reactivecircus.cache4k
+
+import kotlin.test.Test
+import kotlin.test.assertContentEquals
+import kotlin.test.assertNull
+import kotlin.time.Duration.Companion.minutes
+
+private fun eventListOf(vararg elements: CacheEvent<Long, String>) = listOf(elements = elements)
+
+private class TestEventListener : CacheEventListener<Long, String> {
+    val events = mutableListOf<CacheEvent<Long, String>>()
+
+    override fun onEvent(event: CacheEvent<Long, String>) {
+        events.add(event)
+    }
+}
+
+class CacheListenerTest {
+    private val fakeTimeSource = FakeTimeSource()
+
+    @Test
+    fun create() {
+        val eventListener = TestEventListener()
```

Comment:
move this to class member and share?

Reply:
Each test needs a new instance. Of course, we can clear the list of events, but is it necessary to do this instead of simply creating a local object?

Reply:
```kt
class CacheListenerTest {
    private val eventListener = TestEventListener()

    @Test
    fun test1() {
    } 

    @Test
    fun test2() {
    } 
}
```

I believe a new instance of `TestEventListener` will be created for each test.